chainofresponsibilty/src/chainofresponsibilty/handlers/Handler.java

```diff
@@ -0,0 +1,19 @@
+package chainofresponsibilty.handlers;
+
+public abstract class Handler {
+    private Handler next;
+
+    public Handler setNextHandler(Handler next) {
+        this.next = next;
+        return next;
+    }
+
+    public abstract boolean handle(String username, String password);
+
+    protected boolean handleNext(String username, String password) {
```

Comment:
handle next should not be overridden by others. because if anyone forgets to call this method in their implementation chain will break.

Reply:
Done